src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -165,27 +154,6 @@ struct
     ; pending_coinbase_collection: Pending_coinbase.t }
   [@@deriving sexp]
 
-  type serializable =
```

Comment:
Did this code move somewhere or is it no longer needed?

Reply:
[Paul says it's dead code](https://discordapp.com/channels/484437221055922177/502176205211041794/588411290511343626). And if it were used it'd be broken since it calls `Ledger.serializable_of_t` which is a `failwith` stub.